_posts/2024-09-30-security-releases.md

```diff
@@ -0,0 +1,115 @@
+---
+title: September 2024 Security Releases! 
```

Comment:
Can you add the 'description' metadata? The page now supports that function.
```suggestion
title: September 2024 Security Releases! 
description: description
```

Reply:
Added in 33bb348